src/webapp/components/template-selector/TemplateSelector.tsx

```diff
@@ -51,7 +51,7 @@ export const TemplateSelector = ({ settings, themes, onChange }: TemplateSelecto
     const [selectedModel, setSelectedModel] = useState<string>("");
     const [state, setState] = useState<PartialBy<TemplateSelectorState, "type" | "id">>({
         startDate: moment().add("-1", "year").startOf("year"),
-        endDate: moment(),
+        endDate: moment().add("-1", "year").endOf("year"),
```

Comment:
This is a behavior change from previous releases

Now it defaults to last years 01 of January - 31 of December instead of 01 January - Today (next year).